xbmc/network/DNSNameCache.cpp

```diff
@@ -84,20 +109,42 @@ bool CDNSNameCache::Lookup(const std::string& strHostName, std::string& strIpAdd
 #endif
 
   // perform dns lookup
-  struct hostent *host = gethostbyname(strHostName.c_str());
-  if (host && host->h_addr_list[0])
+  struct addrinfo hints;
+  struct addrinfo *result(NULL);
```

Comment:
please use = here for assignment for style reasons. (its a pure C struct - no c'tor)